packages/dom-expressions/src/server.js

```diff
@@ -1,7 +1,26 @@
 import { Aliases, BooleanAttributes, ChildProperties } from "./constants";
 import { sharedConfig, root } from "rxcore";
 import { createSerializer, getLocalHeaderScript } from "./serializer";
-export { createComponent } from "rxcore";
+
+export {
+  getOwner,
+  createComponent,
+  createRenderEffect as effect,
+  createMemo as memo,
+  untrack,
+  mergeProps
+} from "rxcore";
```

Comment:
Is this accurate? I based this off of web/src/core.ts (which I assume indicates which APIs are available server-side, minus any that were already included here. This also matches with what client.js exports.